src/asciidoc/docs/distribute/app-store.adoc

```diff
@@ -0,0 +1,86 @@
+= Listing in the App Store
+:page-slug: app-store
+:page-description: An introduction to the Koji App Store and how to list your Koji app.
+:includespath: ../_includes
+
+After you create, test, and move your app to production, you can share its production URL or list the app in the Koji App Store.
+
+The https://withkoji.com/apps[Koji App Store] is the marketplace for all Koji apps.
+When users edit their Koji profile, they can choose to add an app from the App Store.
+
+By listing in the Koji App Store, you can take advantage of the traffic in the App Store to get exposure for your app.
+Koji also announces noteworthy apps in social media posts.
+
+This topic shows how to create a Koji App Store listing after you move your app to production.
+
+NOTE: You can have multiple listings for the same app.
+
+== Creating an App Store listing
+
+You need the following for your App Store listing:
+
+* A 500x500-pixel icon to be displayed with your listing in the App Store.
+* A 1200x630-pixel thumbnail to be displayed when your listing is shared in other social media sites.
+* Up to 10 videos and screenshots of your app.
+* The production URL of your app.
+To get this URL, go to the app's associated *Project Details*, click *View Published Koji*, and copy the URL from the browser's address bar.
+
+To create your App Store listing,
+
+include::{includespath}/steps-dev-portal.adoc[tag=go2-dev-portal-templates]
+
+. Click *New App Template*.
+
+. Give the app template a name.
++
+This name is not published in the App Store; it is used to identify your listing.
+
+. Fill in the *App Template Details*.
++
+IMPORTANT: All texts and images in your listing must meet https://withkoji.com/resources/terms[Koji's Terms of Use] requirements.
++
+* *Icon* – The 500x500-pixel icon displayed with your listing.
+
+* *Name* – The title of your listing.
+Note that the title could be truncated depending on the width of the browser page; therefore, place distinguishing words near the beginning.
+
+* *Description* – Features and benefits of your app.
+
+* *Share Image* – The 1200x630-pixel thumbnail that is displayed when your app listing is shared in other social media sites.
+
+* *Videos and Screenshots* – Up to 10 videos and screenshots of your app.
+
+* *Bundled App* – The production URL of your app.
+
+* *Price to Remix* – The price (in US dollars) that the creator/influencer pays to publish their customized version of your app.
+Customization is free; creators/influencers are charged only when they publish their customized version in their Koji profile.
+You keep 85% of this amount; Koji gets 15%.
+Leave blank to allow the creator/influencer to publish their customized version for free.
+
+* *Transaction Fee Percent* – The percentage you wish to collect from each transaction if your app allows In-App Purchases.
+You keep 60% of this percentage; Koji gets 40%.
```

Comment:
“Koji gets” sounds a bit awkward to me. We could try leaving this part off, since it seems clear that the remainder are platform fees. Or we could rewrite slightly. Something like, “You keep 85% and the remainder goes toward platform and processor fees.”

Reply:
I think we need a separate page about Koji fees, not hidden in this page. Fees are typically flat amounts, not percentages of a price because our platform and processor expenses are the same cost whether you process $100 or $10. It also contradicts our marketing message that they can create apps *for free*.

I don't think we should call these "fees". We could call them "commissions", but I don't think that's right either.

Regardless, I don't think this topic is the right place to say what the Koji percentage goes to. People intuitively understand it's intended to cover costs and to make a profit for the company.

Reply:
Replaced "gets" with "receives".

Reply:
Good solution! Yes, it would be good to have an explanation of the fees and how developers can make money. There's an entry in the spreadsheet for this topic, so I'll add some notes there. Transaction processing fees are commonplace and usually scale by cost (percentage-based). Where it can be confusing is that the fees are charged to the creators (for remix prices) and to users (for transactions), and then the developers retain a portion of those fees as their revenue.